openslides_backend/action/actions/option/update.py

```diff
@@ -149,3 +179,15 @@ def _get_vote_id(
             if item["value"] == search_value:
                 return key
         return None
+
+    def check_state_change(
+        self, instance: Dict[str, Any], poll: Dict[str, Any]
+    ) -> bool:
+        if poll.get("type") != Poll.TYPE_ANALOG:
+            return False
+        if poll.get("state") != Poll.STATE_CREATED:
```

Comment:
This check has to be removed (analog to poll.create)

Reply:
For analog polls: If the state is created and at least one vote value is given (votesvalid/votesinvalid/votescast), the state must be set to finished.-> Is this wrong then?

Reply:
https://github.com/OpenSlides/OpenSlides/wiki/poll.update